go/src/koding/klientctl/ctlcli/ctlcli.go

```diff
@@ -116,9 +116,10 @@ func ExitErrAction(f ExitingErrCommand, log logging.Logger, cmdName string) cli.
 		}
 
 		Close()
-		ExitFunc(exit)
```

Comment:
This is the only place where we return non 0-1  return code in klientctl https://github.com/koding/koding/blob/master/go/src/koding/klientctl/logcmd/command.go#L90